src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -230,7 +230,7 @@ public function addXmlContent($content, $charset = 'UTF-8')
         $dom->validateOnParse = true;
 
         if ('' !== trim($content)) {
-            @$dom->loadXML($content, LIBXML_NONET);
+            @$dom->loadXML($content, LIBXML_NONET | LIBXML_PARSEHUGE);
```

Comment:
Manual says that it only relaxes any hardcoded limit from the parser.
https://secure.php.net/manual/en/libxml.constants.php

It only for Libxml >= 2.7.0 but I dont know is version below is widespread.
For ex. on CentOS 6 is 2.7.6